data/blog/electron-13-0.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Electron 13.0.0
+author:
+- sofianguy
+- georgexu99
+- VerteDinde
+date: '2021-05-25'
+---
+
+Electron 13.0.0 has been released! It includes upgrades to Chromium `91`, V8 `9.1` and Node.js `14.16.0`. We've added several API updates, bug fixes, and general improvements. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 13.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). Continue reading for details about this release, and please share any feedback you have!
+
+## Notable Changes
+
+### Stack Changes
+
+* Chromium `91`
+    * [New in Chrome 91](https://developer.chrome.com/blog/new-in-chrome-91/)
+    * [New in Chrome 90](https://developer.chrome.com/blog/new-in-chrome-90/)
+* Node.js `14.16.0`
+    * [Node 14.16.0 blog post](https://nodejs.org/en/blog/release/v14.16.0/)
+    * [Node 14.0.0 blog post](https://nodejs.org/en/blog/release/v14.0.0/)
+* V8 `9.1`
+    * [V8 9.1 blog post](https://v8.dev/blog/v8-release-91)
+    * [V8 9.0 blog post](https://v8.dev/blog/v8-release-90)
+
+### Highlight Features
+
+* Added `process.contextIsolated` property that indicates whether the current renderer context has `contextIsolation` enabled. [#28252](https://github.com/electron/electron/pull/28252) 
+* Added new `session.storagePath` API to get the path on disk for session-specific data. [#28866](https://github.com/electron/electron/pull/28866) 
+* Deprecated the `new-window` event of `WebContents`. It is replaced by `webContents.setWindowOpenHandler()`
+* The `submitURL` option for `crashReporter.start` is no longer a required argument when `uploadToServer` is false. [#28283](https://github.com/electron/electron/pull/28283)
```

Comment:
This one feels like more of a bugfix and not up at the same level as the other items in this section?

Reply:
Yeah, true - I'll remove this one